vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -88,21 +90,19 @@ AdsImpl::AdsImpl(AdsClient* ads_client)
     : is_foreground_(false),
       active_tab_id_(0),
       next_easter_egg_timestamp_in_seconds_(0),
-      client_(std::make_unique<Client>(this, ads_client)),
-      bundle_(std::make_unique<Bundle>(this, ads_client)),
-      ads_serve_(std::make_unique<AdsServe>(this, ads_client, bundle_.get())),
+      client_(std::make_unique<Client>(this)),
+      bundle_(std::make_unique<Bundle>(this)),
+      ads_serve_(std::make_unique<AdsServe>(this, bundle_.get())),
       subdivision_targeting_(std::make_unique<
           SubdivisionTargeting>(ads_client)),
-      frequency_capping_(std::make_unique<FrequencyCapping>(client_.get())),
-      ad_conversions_(std::make_unique<AdConversions>(
-          this, ads_client, client_.get())),
-      page_classifier_(std::make_unique<PageClassifier>(this)),
+      ad_conversions_(std::make_unique<AdConversions>(this)),
+      page_classifier_(std::make_unique<classification::PageClassifier>(this)),
       purchase_intent_classifier_(std::make_unique<PurchaseIntentClassifier>(
           kPurchaseIntentSignalLevel, kPurchaseIntentClassificationThreshold,
```

Comment:
should purchase intent sit under the `classification::` namespace as well?

Reply:
Fixed